src/components/Tree/TreeItem/TreeItem.tsx

```diff
@@ -91,6 +91,12 @@ export const TreeItem = memo(
                 (typeof over?.data?.current?.accepts === 'string' &&
                     over.data.current.accepts?.split(', ').includes(active.data.current.type)));
 
+        useEffect(() => {
```

Comment:
This is not the place to fix this. You have onDragStart handler where you can set cursor to grabbing

Reply:
it is not about dragging, it's about `no-drag`, when the item is over a not allowed place to be dropped.


https://github.com/Frontify/fondue/assets/357211/5466329a-0833-42eb-bebd-f90a15e3fee3

Reply:
Ah okay,  still no need for duplicating logic, you have the onDragOver method on useDndMonitor.

Reply:
Ok, i moved it to the `onDragMove` method. `onDragOver` cannot be used for this because it ONLY tracks when the draggable moves from one item position to another and not when the draggable gets moved at the different levels.

see video (using `onDragOver`):

https://github.com/Frontify/fondue/assets/357211/33ee5e23-3c1f-4603-8f50-dcb5cbf6bfbc